triggers/tests/BUILD.bazel

```diff
@@ -184,3 +183,56 @@ da_scala_library(
     for lf_version in DAML_LF_VERSIONS
     for suffix in [("-" + lf_version) if lf_version else ""]
 ]
+
+[
+    da_scala_test_suite(
+        name = "trigger-failure-integration-tests" + suffix,
+        srcs = [
+            "src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/FailureTesting.scala",
```

Comment:
Separate testing of FailureTesting.scala for greater build/CI control of this expensive test